### ISSUE TITLE
Cloudwatch Alarm Date formatting causes terraform to get stuck in a loop

### ISSUE THREAD

**Issue by kazeshini178** [localstack/localstack#3062]
# Type of request: This is a ...
[x] bug report
[ ] feature request

# Detailed description
When creating a cloud watch alarm, in my case linked to SQS. Localstack (0.11.5) returns 
```xml
<DescribeAlarmsResponse xmlns="http://monitoring.amazonaws.com/doc/2010-08-01/">
    <DescribeAlarmsResult>
        <MetricAlarms>
            <member>
                <ActionsEnabled>true</ActionsEnabled>
                <AlarmActions>
                    <member>arn:aws:sns:eu-west-1:000000000000:localstack-dead-letter-alert-topic</member>
                </AlarmActions>
                <AlarmArn></AlarmArn>
                <AlarmConfigurationUpdatedTimestamp>2020-10-02 13:40:59.239247</AlarmConfigurationUpdatedTimestamp>
                <AlarmDescription>Alert that there are records that have failed to be processed</AlarmDescription>
                <AlarmName>localstack-queue-alarm</AlarmName>
                <ComparisonOperator>GreaterThanOrEqualToThreshold</ComparisonOperator>
                <Dimensions>
                    <member>
                        <Name>QueueName</Name>
                        <Value>localstack-dead-letter-queue</Value>
                    </member>
                </Dimensions>
                <EvaluationPeriods>1</EvaluationPeriods>
                <InsufficientDataActions></InsufficientDataActions>
                <MetricName>ApproximateNumberOfMessagesVisible</MetricName>
                <Namespace>AWS/SQS</Namespace>
                <OKActions></OKActions>
                <Period>60</Period>
                <StateReason></StateReason>
                <StateReasonData>{}</StateReasonData>
                <StateUpdatedTimestamp>2020-10-02 13:40:59.239250</StateUpdatedTimestamp>
                <StateValue>OK</StateValue>
                <Statistic>Sum</Statistic>
                <Threshold>1</Threshold>
                <Unit>None</Unit>
            </member>
        </MetricAlarms>
    </DescribeAlarmsResult>
</DescribeAlarmsResponse>
```
The dates in this response cause terraform to error, preventing it from confirming that the resource has been created in localstack. It seems terraform expect the date to follow ISO 8601
i.e: 2020-10-02 13:40:59.239247 -> 2020-10-02T13:40:59.239247

## Expected behavior
Date to be in expected format and match AWS date response format.
```bash
aws cloudwatch describe-alarms --alarm-names euw1-queue-alarm 
```
```json
{
    "MetricAlarms": [
        {
            "AlarmName": "euw1-queue-alarm ",
            "AlarmDescription": "Alert that there are records that have failed to be processed",
            "AlarmConfigurationUpdatedTimestamp": "2020-08-21T12:55:45.009Z",
            "ActionsEnabled": true,
            "OKActions": [],
            "AlarmActions": [
                "arn:aws:sns:eu-west-1:003518887679:euw1-dead-letter-alert-topic"
            ],
            "InsufficientDataActions": [],
            "StateValue": "OK",
            "StateReason": "Threshold Crossed: 1 out of the last 1 datapoints [0.0 (23/09/20 11:58:00)] was not greater than or equal to the threshold (1.0) (minimum 1 datapoint for ALARM -> OK transition).",
            "StateUpdatedTimestamp": "2020-09-23T11:59:41.471Z",
            "MetricName": "ApproximateNumberOfMessagesVisible",
            "Namespace": "AWS/SQS",
            "Statistic": "Sum",
            "Dimensions": [
                {
                    "Name": "QueueName",
                    "Value": "euw1-dead-letter-queue"
                }
            ],
            "Period": 60,
            "EvaluationPeriods": 1,
            "DatapointsToAlarm": 1,
            "Threshold": 1.0,
            "ComparisonOperator": "GreaterThanOrEqualToThreshold",
            "TreatMissingData": "missing"
        }
    ]
}
```

## Actual behavior

Date formatted incorrectly/different to expected format and aws's return format.
```log
aws_cloudwatch_metric_alarm.sns_dlq_queue_alarm: Still creating... [2m50s elapsed]
2020-10-02T15:43:46.967+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: 2020/10/02 15:43:46 [DEBUG] [aws-sdk-go] DEBUG: Retrying Request monitoring/DescribeAlarms, attempt 122020-10-02T15:43:46.967+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: 2020/10/02 15:43:46 [DEBUG] [aws-sdk-go] DEBUG: Request monitoring/DescribeAlarms Details:
2020-10-02T15:43:46.967+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: ---[ REQUEST POST-SIGN ]-----------------------------
2020-10-02T15:43:46.967+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: POST / HTTP/1.1
2020-10-02T15:43:46.967+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: Host: localhost:4566
2020-10-02T15:43:46.967+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: User-Agent: aws-sdk-go/1.34.21 (go1.14.5; windows; amd64) APN/1.0 HashiCorp/1.0 Terraform/0.12.29 (+https://www.terraform.io)
2020-10-02T15:43:46.967+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: Content-Length: 99
2020-10-02T15:43:46.967+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: Authorization: AWS4-HMAC-SHA256 Credential=mock_access_key/20201002/eu-west-1/monitoring/aws4_request, SignedHeaders=content-length;content-type;host;x-amz-date, Signature=20b854633c3af47e28e2c435f3fe5eb71df15e9457424708793c0edf3a4fdece
2020-10-02T15:43:46.967+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: Content-Type: application/x-www-form-urlencoded; charset=utf-8
2020-10-02T15:43:46.967+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: X-Amz-Date: 20201002T134346Z
2020-10-02T15:43:46.967+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: Accept-Encoding: gzip
2020-10-02T15:43:46.967+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe:
2020-10-02T15:43:46.967+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: Action=DescribeAlarms&AlarmNames.member.1=localstack-queue-alarm&Version=2010-08-01   
2020-10-02T15:43:46.967+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: -----------------------------------------------------
2020-10-02T15:43:46.994+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: 2020/10/02 15:43:46 [DEBUG] [aws-sdk-go] DEBUG: Response monitoring/DescribeAlarms Details:
2020-10-02T15:43:46.995+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: ---[ RESPONSE ]--------------------------------------
2020-10-02T15:43:46.995+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: HTTP/1.1 200
2020-10-02T15:43:46.995+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: Connection: close
2020-10-02T15:43:46.995+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: Content-Length: 1277
2020-10-02T15:43:46.995+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: Access-Control-Allow-Headers: authorization,content-type,content-md5,cache-control,x-amz-content-sha256,x-amz-date,x-amz-security-token,x-amz-user-agent,x-amz-target,x-amz-acl,x-amz-version-id,x-localstack-target,x-amz-tagging
2020-10-02T15:43:46.995+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: Access-Control-Allow-Methods: HEAD,GET,PUT,POST,DELETE,OPTIONS,PATCH
2020-10-02T15:43:46.995+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: Access-Control-Allow-Origin: *
2020-10-02T15:43:46.995+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: Access-Control-Expose-Headers: x-amz-version-id
2020-10-02T15:43:46.995+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: Content-Type: text/html; charset=utf-8
2020-10-02T15:43:46.995+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: Date: Fri, 02 Oct 2020 13:43:46 GMT
2020-10-02T15:43:46.995+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: Server: hypercorn-h11
2020-10-02T15:43:46.995+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: X-Amzn-Requestid: SI6V2OSKOT741JF971ZJ0CSW8WZLZA25YLX9WSL6L2XH7BJ7D6IW
2020-10-02T15:43:46.995+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe:
2020-10-02T15:43:46.995+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe:
2020-10-02T15:43:46.995+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: -----------------------------------------------------
ws-sdk-go] <DescribeAlarmsResponse xmlns="http://monitoring.amazonaws.com/doc/2010-08-01/"><DescribeAlarmsResult><MetricAlarms><member><ActionsEnabled>true</ActionsEnabled><AlarmActions><member>arn:aws:sns:eu-west-1:000000000000:localstack-dead-letter-alert-topic</member></AlarmActions><AlarmArn></AlarmArn><AlarmConfigurationUpdatedTimestamp>2020-10-02 13:40:59.239247</AlarmConfigurationUpdatedTimestamp><AlarmDescription>Alert that there are records that have failed to be processed</AlarmDescription><AlarmName>localstack-update-queue-alarm</AlarmName><ComparisonOperator>GreaterThanOrEqualToThreshold</ComparisonOperator><Dimensions><member><Name>QueueName</Name><Value>localstack-update-dead-letter-queue</Value></member></Dimensions><EvaluationPeriods>1</EvaluationPeriods><InsufficientDataActions></InsufficientDataActions><MetricName>ApproximateNumberOfMessagesVisible</MetricName><Namespace>AWS/SQS</Namespace><OKActions></OKActions><Period>60</Period><StateReason></StateReason><StateReasonData>{}</StateReasonData><StateUpdatedTimestamp>2020-10-02 13:40:59.239250</StateUpdatedTimestamp><StateValue>OK</StateValue><Statistic>Sum</Statistic><Threshold>1</Threshold><Unit>None</Unit></member></MetricAlarms></DescribeAlarmsResult></DescribeAlarmsResponse>
2020-10-02T15:43:46.995+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: 2020/10/02 15:43:46 [DEBUG] [aws-sdk-go] DEBUG: Unmarshal Response monitoring/DescribeAlarms failed, attempt 12/25, error SerializationError: failed decoding Query response
2020-10-02T15:43:46.995+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe:                                                                                                      status code: 200, request id: SI6V2OSKOT741JF971ZJ0CSW8WZLZA25YLX9WSL6L2XH7BJ7D6IW
2020-10-02T15:43:46.995+0200 [DEBUG] plugin.terraform-provider-aws_v3.7.0_x5.exe: caused by: parsing time "2020-10-02 13:40:59.239247" as "2006-01-02T15:04:05.999999999Z": cannot parse " 13:40:59.239247" as "T"
```

# Steps to reproduce

## Command used to start LocalStack

`docker run localstack/localstack:0.11.5 -p 4566:4566`

**Comment by whummer** [comment]
Hi @kazeshini178 , thanks for reporting. This should be fixed in #3110 - can you please pull the `latest` Docker image and give it another try? Please report here if the problem persists. Thanks

**Comment by kazeshini178** [comment]
@whummer Thanks seems the date is has been sorted out, though seems that their isnt an ARN that getting assigned to the created alarm. Which then causes
```
Error: error listing tags for CloudWatch Metric Alarm (): InvalidParameter: 1 validation error(s) found.
- minimum field size of 1, ListTagsForResourceInput.ResourceARN.
```
But can log that as a separate issue.
Look forward to the release with this fix in it 😄
